src/chainparams.cpp

```diff
@@ -298,3 +320,24 @@ bool SelectParamsFromCommandLine()
     SelectParams(network);
     return true;
 }
+
+
+// Block height must be >0 and less than the halving interval block height
+// Index variable i ranges from 0 - (vFoundersRewardAddress.size()-1)
+std::string CChainParams::GetFoundersRewardAddress(size_t nHeight) const
+{
+    assert(nHeight>0 && nHeight<consensus.nSubsidyHalvingInterval);
+    size_t i = (size_t)floor((double(nHeight)/consensus.nSubsidyHalvingInterval)*vFoundersRewardAddress.size());
```

Comment:
Looks like this is definitely a bug based on:

"The end result (as shown in the diagram) is that there will ultimately be 21 million ⓩ, and 10% of it, or 2.1 million ⓩ, will have been initially distributed to the founders."